convert2rhel/actions/pre_ponr_changes/backup_system.py

```diff
@@ -61,3 +63,93 @@ def run(self):
 
         repo.backup_yum_repos()
         repo.backup_varsdir()
+
+
+class BackupPackageFiles(actions.Action):
+    id = "BACKUP_PACKAGE_FILES"
+
+    def run(self):
+        """Backup changed package files"""
+        loggerinst.task("Prepare: Backup package files")
+
+        super(BackupPackageFiles, self).run()
+
+        backup.package_files_changes = self._get_rpm_va()
+
+        for file in backup.package_files_changes:
+            if file["status"] == "missing":
+                pass  # no action needed now
+            elif re.search(r"[5]", file["status"]):
+                # If the MD5 checksum differs, the content of the file differs
+                file["backup"] = backup.RestorableFile(file["path"])
+                file["backup"].backup()
+
+    def _get_rpm_va(self):
+        """Run the rpm -Va command to get changes made to package files.
+        Return them as a list of dict, for example:
+        [{"status":"S5T", "file_type":"c", "path":"/etc/yum.repos.d/CentOS-Linux-AppStream.repo"}]
+        """
+        cmd = ["rpm", "-Va"]
```

Comment:
My opinion about deprecating `--no-rpm-va`:

I think this will greatly impact in the wait time for the conversion in some use cases where the user has lots and lots of files, modifications to the system etc... Maybe the last suggestion would be more ideal, i.e: "We could make this Action depend on --no-rpm-va not being set.". 

+1 to parse the initial output of `rpm -Va` instead of running it a second time. I don't think the parsing needs to be there, it could be in this action as long as it checks if there is anything to parse first. Seems to me that the only case where we care about parsing the contents of that output is here (correct me if I'm wrong).

Reply:
@abadger, great points. Indeed we should not run the `rpm -Va` twice as it is super slow. We shall reuse the output of the first invocation.

I'd vote for keeping the --no-rpm-va switch functionality, i.e. it would prevent running rpm -Va, but because that can lead to an incomplete rollback I'd newly require the user to set the `CONVERT2RHEL_INCOMPLETE_ROLLBACK=1` env var (after #683 is merged). What do you think, @r0x0d, @abadger?

The second best option for me would be to _"change the help documentation to note that --no-rpm-va only disables the rpm -Va which is run after the conversion"_.

Reply:
+1 to that, @bocekm